<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>EinstellungenWindow</class>
 <widget class="QMainWindow" name="EinstellungenWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>700</width>
    <height>702</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Einstellungen</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout_3">
    <item>
     <widget class="QWidget" name="widget" native="true">
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <widget class="QTabWidget" name="tab_widget">
         <property name="currentIndex">
          <number>4</number>
         </property>
         <widget class="QWidget" name="anst_tab">
          <property name="enabled">
           <bool>true</bool>
          </property>
          <attribute name="title">
           <string>Anschlussstellen</string>
          </attribute>
          <layout class="QVBoxLayout" name="verticalLayout_15">
           <item>
            <widget class="QSplitter" name="anst_splitter">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <widget class="QTableView" name="agl_table_view">
              <property name="alternatingRowColors">
               <bool>true</bool>
              </property>
              <property name="selectionMode">
               <enum>QAbstractItemView::MultiSelection</enum>
              </property>
              <property name="selectionBehavior">
               <enum>QAbstractItemView::SelectRows</enum>
              </property>
              <property name="sortingEnabled">
               <bool>true</bool>
              </property>
             </widget>
             <widget class="QWidget" name="anst_control_widget" native="true">
              <layout class="QVBoxLayout" name="verticalLayout_10">
               <item>
                <widget class="QGroupBox" name="agl_group">
                 <property name="title">
                  <string>Anschlussgleis (Agl)</string>
                 </property>
                 <layout class="QVBoxLayout" name="verticalLayout_11">
                  <item>
                   <widget class="QComboBox" name="agl_combo">
                    <property name="editable">
                     <bool>true</bool>
                    </property>
                   </widget>
                  </item>
                  <item>
                   <widget class="QPushButton" name="agl_filter_button">
                    <property name="text">
                     <string>Filter</string>
                    </property>
                   </widget>
                  </item>
                  <item>
                   <widget class="QPushButton" name="anst_ungroup_button">
                    <property name="text">
                     <string>Neue Anst</string>
                    </property>
                   </widget>
                  </item>
                 </layout>
                </widget>
               </item>
               <item>
                <widget class="QGroupBox" name="anst_group">
                 <property name="title">
                  <string>Anschlussstelle (Anst)</string>
                 </property>
                 <layout class="QVBoxLayout" name="verticalLayout_14">
                  <item>
                   <widget class="QComboBox" name="anst_combo">
                    <property name="editable">
                     <bool>true</bool>
                    </property>
                   </widget>
                  </item>
                  <item>
                   <widget class="QPushButton" name="anst_group_button">
                    <property name="text">
                     <string>Zuordnen</string>
                    </property>
                   </widget>
                  </item>
                  <item>
                   <widget class="QPushButton" name="anst_rename_button">
                    <property name="text">
                     <string>Umbenennen</string>
                    </property>
                   </widget>
                  </item>
                 </layout>
                </widget>
               </item>
               <item>
                <spacer name="verticalSpacer_2">
                 <property name="orientation">
                  <enum>Qt::Vertical</enum>
                 </property>
                 <property name="sizeHint" stdset="0">
                  <size>
                   <width>20</width>
                   <height>40</height>
                  </size>
                 </property>
                </spacer>
               </item>
              </layout>
             </widget>
            </widget>
           </item>
          </layout>
         </widget>
         <widget class="QWidget" name="bahnhof_tab">
          <property name="enabled">
           <bool>true</bool>
          </property>
          <attribute name="title">
           <string>Bahnh√∂fe</string>
          </attribute>
          <layout class="QVBoxLayout" name="verticalLayout_8">
           <item>
            <widget class="QSplitter" name="bf_splitter">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <widget class="QTableView" name="gl_table_view">
              <property name="alternatingRowColors">
               <bool>true</bool>
              </property>
              <property name="selectionMode">
               <enum>QAbstractItemView::MultiSelection</enum>
              </property>
              <property name="selectionBehavior">
               <enum>QAbstractItemView::SelectRows</enum>
              </property>
              <property name="sortingEnabled">
               <bool>true</bool>
              </property>
             </widget>
             <widget class="QWidget" name="bf_control_widget" native="true">
              <layout class="QVBoxLayout" name="verticalLayout">
               <item>
                <widget class="QGroupBox" name="gl_group">
                 <property name="title">
                  <string>Gleis (Gl)</string>
                 </property>
                 <layout class="QVBoxLayout" name="verticalLayout_7">
                  <item>
                   <widget class="QComboBox" name="gl_combo">
                    <property name="editable">
                     <bool>true</bool>
                    </property>
                   </widget>
                  </item>
                  <item>
                   <widget class="QPushButton" name="gl_filter_button">
                    <property name="text">
                     <string>Filter</string>
                    </property>
                   </widget>
                  </item>
                  <item>
                   <widget class="QPushButton" name="bs_ungroup_button">
                    <property name="text">
                     <string>Neuer Bahnsteig</string>
                    </property>
                   </widget>
                  </item>
                 </layout>
                </widget>
               </item>
               <item>
                <widget class="QGroupBox" name="bs_group">
                 <property name="title">
                  <string>Bahnsteig (Bs)</string>
                 </property>
                 <layout class="QVBoxLayout" name="verticalLayout_4">
                  <item>
                   <widget class="QComboBox" name="bs_combo">
                    <property name="editable">
                     <bool>true</bool>
                    </property>
                   </widget>
                  </item>
                  <item>
                   <widget class="QPushButton" name="bs_group_button">
                    <property name="text">
                     <string>Zuordnen</string>
                    </property>
                   </widget>
                  </item>
                  <item>
                   <widget class="QPushButton" name="bs_rename_button">
                    <property name="text">
                     <string>Umbenennen</string>
                    </property>
                   </widget>
                  </item>
                 </layout>
                </widget>
               </item>
               <item>
                <widget class="QGroupBox" name="bft_group">
                 <property name="title">
                  <string>Bahnhofteil (Bft)</string>
                 </property>
                 <layout class="QVBoxLayout" name="verticalLayout_5">
                  <item>
                   <widget class="QComboBox" name="bft_combo">
                    <property name="editable">
                     <bool>true</bool>
                    </property>
                   </widget>
                  </item>
                  <item>
                   <widget class="QPushButton" name="bft_rename_button">
                    <property name="text">
                     <string>Umbenennen</string>
                    </property>
                   </widget>
                  </item>
                  <item>
                   <widget class="QPushButton" name="bf_ungroup_button">
                    <property name="text">
                     <string>Neuer Bahnhof</string>
                    </property>
                   </widget>
                  </item>
                 </layout>
                </widget>
               </item>
               <item>
                <widget class="QGroupBox" name="bf_group">
                 <property name="title">
                  <string>Bahnhof (Bf)</string>
                 </property>
                 <layout class="QVBoxLayout" name="verticalLayout_6">
                  <item>
                   <widget class="QComboBox" name="bf_combo">
                    <property name="editable">
                     <bool>true</bool>
                    </property>
                   </widget>
                  </item>
                  <item>
                   <widget class="QPushButton" name="bf_group_button">
                    <property name="text">
                     <string>Zuordnen</string>
                    </property>
                   </widget>
                  </item>
                  <item>
                   <widget class="QPushButton" name="bf_rename_button">
                    <property name="text">
                     <string>Umbenennen</string>
                    </property>
                   </widget>
                  </item>
                 </layout>
                </widget>
               </item>
               <item>
                <spacer name="verticalSpacer">
                 <property name="orientation">
                  <enum>Qt::Vertical</enum>
                 </property>
                 <property name="sizeHint" stdset="0">
                  <size>
                   <width>20</width>
                   <height>40</height>
                  </size>
                 </property>
                </spacer>
               </item>
              </layout>
             </widget>
            </widget>
           </item>
          </layout>
         </widget>
         <widget class="QWidget" name="strecken_tab">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <attribute name="title">
           <string>Strecken</string>
          </attribute>
         </widget>
         <widget class="QWidget" name="zugschema_tab">
          <attribute name="title">
           <string>Zugschema</string>
          </attribute>
          <layout class="QHBoxLayout" name="horizontalLayout_2">
           <item>
            <layout class="QVBoxLayout" name="verticalLayout_2">
             <item>
              <widget class="QLabel" name="zugschema_name_label">
               <property name="text">
                <string>Zugschema</string>
               </property>
               <property name="buddy">
                <cstring>zugschema_name_label</cstring>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QComboBox" name="zugschema_name_combo"/>
             </item>
             <item>
              <widget class="QLabel" name="zugschema_details_label">
               <property name="text">
                <string>Kategorien</string>
               </property>
               <property name="buddy">
                <cstring>zugschema_details_table</cstring>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QTableView" name="zugschema_details_table">
               <property name="selectionBehavior">
                <enum>QAbstractItemView::SelectRows</enum>
               </property>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </widget>
         <widget class="QWidget" name="hilfe_tab">
          <attribute name="title">
           <string>Hilfe</string>
          </attribute>
          <layout class="QVBoxLayout" name="verticalLayout_9">
           <item>
            <widget class="QTextEdit" name="hilfe_text">
             <property name="readOnly">
              <bool>true</bool>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </widget>
       </item>
       <item>
        <widget class="QDialogButtonBox" name="dialog_button_box">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="standardButtons">
          <set>QDialogButtonBox::Apply|QDialogButtonBox::Cancel|QDialogButtonBox::Ok|QDialogButtonBox::Reset</set>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>dialog_button_box</sender>
   <signal>accepted()</signal>
   <receiver>EinstellungenWindow</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>dialog_button_box</sender>
   <signal>rejected()</signal>
   <receiver>EinstellungenWindow</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>accept()</slot>
  <slot>reject()</slot>
 </slots>
</ui>
